packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -15,6 +15,7 @@ import {Checkbox} from '@react-spectrum/checkbox';
 import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
 import {DOMRef} from '@react-types/shared';
 import {FocusRing, useFocusRing} from '@react-aria/focus';
+import {gridMap} from '@react-aria/grid/src/utils';
```

Comment:
I figured you already know, but FYI this won't work once it is built due to importing from src

Reply:
o dear, i didn't notice my IDE did that, thank you